src/hotspot/share/utilities/globalDefinitions.hpp

```diff
@@ -417,6 +418,9 @@ const uintx max_uintx = (uintx)-1;
 
 typedef unsigned int uint;   NEEDS_CLEANUP
 
+//----------------------------------------------------------------------------------------------------
```

Comment:
If you look at the file there's a bunch of sections starting with dashes. I don't think you need a new section for a single typedef
```suggestion
```